content/php-client/exception.md

```diff
@@ -0,0 +1,128 @@
+    $responseMessage = $e->getResponseMessage();
+    $responseCode = $e->getResponseCode();
+}
+```
+
+ 
+Two types of exception inherits from this exception: server exception and client exception.
+
+## Server exception
+
+A server exception thrown by the client means that the server failed to fulfill an apparently valid request.
+It's an HTTP code in the family 5xx.
+
+This exception is `Akeneo\Pim\Exception\ServerErrorHttpException`.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (ServerHttpException $e) {
+    // do your stuff with the exception
+    $requestBody = $e->getRequest()->getBody();
+    $responseBody = $e->getResponse()->getBody();
+    $responseMessage = $e->getResponseMessage();
+    $responseCode = $e->getResponseCode();
+}
+```
+
+## Client exception
+
+A server exception could be thrown by the client and means is intended for situations in which the client seems to have errored.
+It's an HTTP code in the family 4xx.
+
+This exception is `Akeneo\Pim\Exception\ClientErrorHttpException`. It is subdivided in several specific exceptions.
+
+### Bad request exception
+
+This exception could be thrown if the request is not a valid JSON. It corresponds to the HTTP code 400.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (BadRequestHttpException $e) {
+    // do your stuff with the exception
+    $requestBody = $e->getRequest()->getBody();
+    $responseBody = $e->getResponse()->getBody();
+    $responseMessage = $e->getResponseMessage();
+    $responseCode = $e->getResponseCode();
+}
+```
+
+::: info
+It should not occurred with the PHP client, because the JSON is generated from PHP array.
+:::
+
+### Not found exception
+
+This exception could be thrown if the requested resource does not exist. It corresponds to the HTTP code 404.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (NotFoundHttpException $e) {
+    // do your stuff with the exception
+    $requestBody = $e->getRequest()->getBody();
+    $responseBody = $e->getResponse()->getBody();
+    $responseMessage = $e->getResponseMessage();
+    $responseCode = $e->getResponseCode();
+}
+```
+
+### Unauthorized exception
+
+This exception could be thrown if you don't have the permission to access to the resource. It corresponds to the HTTP code 401.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (UnauthorizedHttpException $e) {
+    // do your stuff with the exception
+    $requestBody = $e->getRequest()->getBody();
+    $responseBody = $e->getResponse()->getBody();
+    $responseMessage = $e->getResponseMessage();
+    $responseCode = $e->getResponseCode();
+}
+```
+
+::: info
+It should not occurred as the PHP client automatically authenticates the request for you.
+:::
+
+### Unprocessable entity exception
+
+This exception could be thrown if the data are not valid. In this exception, an array of errors could be returned.
+It returns an empty array if there is no error in the array.
```

Comment:
"if there's no error" is too ambiguous. We are in an exception so there is an error, but the array describing those errors is not mandatory (it depends).